soh/soh/Enhancements/mods.cpp

```diff
@@ -1069,6 +1069,17 @@ void RegisterRandomizedEnemySizes() {
         }
 
         Actor_SetScale(actor, actor->scale.z * randomScale);
+
+        if (CVarGetInteger("gEnemySizeScalesHealth", 0) && (actor->category == ACTORCAT_ENEMY)) {
+            // Scale the health based on a smaller factor than randomScale
+            float healthScalingFactor = 0.8f; // Adjust this factor as needed
```

Comment:
How much has this been playtested to make sure it makes sense?

I imagine the smallest enemy should probably be a one hit kill, whereas the largest probably should have like 3x the amount of health?

Reply:
Bigger doesn't always mean tougher though.

Reply:
I have tested it a bit, but I will do more testing today. 

And bigger does mean tougher in this PR. That's the whole point..

Reply:
> I have tested it a bit, but I will do more testing today.
> 
> And bigger does mean tougher in this PR. That's the whole point..

Hahaha, you do have a point there. Make the biggest Stalchild Bloodborne tough.